src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -2801,7 +2801,7 @@ namespace EvaporativeCoolers {
         auto &EvapCond(state.dataEvapCoolers->EvapCond);
 
         EvapCoolIndex = int(Par(1));
-        DryOrWetOperatingMode = OperatingMode(Par(2));
+        DryOrWetOperatingMode = OperatingMode(int(Par(2)));
```

Comment:
A compile error occurs using Visual Studio 2017. "int" may not be needed for other MSVS versions. It depends on reviewers to determine if "int" is kept or not.

Reply:
It's not great, but we need to take a wholehearted refactor on the solve-root stuff to modernize it.  C++ will offer new ways to marshal data in and out of the optimization routines that were not available in Fortran.  For today, if this works, then so be it.